.github/settings.yml

```diff
@@ -93,6 +103,11 @@ labels:
   - name: released
     color: A9A9A9
     description: Completed work has been released
+  - name: not-reviewed
+    cat: default
+    color: '222222'
+    description: Issue has not been reviewed by Auro team member
+    process: 'Automated: auto-applied with new issue, manually removed by reviewer'
```

Comment:
This should not be here. It appears that you merged this commit into the last commit on the repo.